olympics.md

```diff
@@ -0,0 +1,25 @@
+--
+title: All the medal winners in the history of the Summer Olympic Games and more 
+description: A compilation of all the athletes who won at least one medal in every SOG, from Athens 1896 to Paris 2024, with emphasis on the most recent Games
+keywords: olympics, olympic games, paris 2024, medal winners
+date: 01/10/2024
+modified: 01/10/2024
+--
+
+## Insights and curated datasets
+
+- [All the medal winners in the history of the Summer Olympic Games until Tokyo 2020](https://datahub.io/@cheredia19/all-medallists-until-tokyo-2020)
+- [All athletes who participated at the Paris 2024 Olympic Games and some curiosities about them](https://datahub.io/@cheredia19/paris-2024-osg-athletes)
+- [All the individual and team medal winners in Paris 2024 and more](https://datahub.io/@cheredia19/paris-2024-medals-and-medallists)
+- [Regions and continents with the most medals at the Paris 2024 Olympic Games](https://datahub.io/@cheredia19/paris-2024-olympics)
+
+## Sources
+
+- [Kaggle - Olympic Summer & Winter Games, 1896-2022](https://www.kaggle.com/datasets/piterfm/olympic-games-medals-19862018?select=olympic_medals.csv)
+- [Kaggle - Paris 2024 Olympic Summer Games](https://www.kaggle.com/datasets/piterfm/paris-2024-olympic-summer-games)
+
+Data was originally scraped from olympics.com.
+
+## Data license
```

Comment:
not sure about this because licenses are per dataset and should be on dataset showcase page